### ISSUE TITLE
LBA1 Voice quality cleanup

### ISSUE THREAD

**Issue by elmuerte** [LBALab/lba2remake#695]
**Is your feature request related to a problem? Please describe.**
The LBA1 voice samples have a lot of noise. The main cause for this is the original Creative Voice file codec's limitation.
As this remake converts the data using ffmpeg the result can be cleaned up.

**Describe the solution you'd like**

Add the following filter to the ffmpeg command `-af afftdn,anlmdn=s=7:p=0.002:r=0.002:m=15` in https://github.com/LBALab/lba2remake/blob/master/utils/convert/convert.ts#L317

It removes a lot of the static noise.

**Describe alternatives you've considered**
Additionally, for LBA1's voice samples Vorbis provides excellent compression ratio at the low frequency of the original VOC files without noticeable quality loss (e.g. 11Hz mono). Comparable AAC LC was bigger and added more nasal sound to it.

**Additional context**
Result after using the filter, encoding it as Vorbis in a WebM container: 
[059_001.webm](https://github.com/LBALab/lba2remake/assets/130263/8c5712dd-82e5-4bde-9d06-59ad49147d74)
The original: 
[059_001-nofilter.webm](https://github.com/LBALab/lba2remake/assets/130263/093b24f4-523c-4b46-9aa3-a3e5d3a09d64)

Note, ffmpeg cannot be used for the conversion to Vorbis, but it can be used to copy the vorbis stream from an Ogg container to a WebM container. Vorbis encoding was done with `q5`.

**Comment by xesf** [comment]
We have a similar de-noised function in another project here: https://github.com/LBALab/lba-noise-reduction/blob/master/src/convert.ts#L229 (this was actually created to be fed into Classic LBA version to replace the Vorbis files but 2.21 never had a chance to replace them. There was some small developments required)

It uses a slightly different command line. I think I had something like yours before but we removed when we created this version. Maybe it makes sense to apply your changes on both projects. LBA2remake for convenience and the de-noised repo for other needs as it does Samples as well.
